src/Reflection/ReflectionObject.php

```diff
@@ -66,30 +64,6 @@ public static function export($instance = null) : string
     }
 
     /**
-     * Cannot instantiate this way, use ReflectionObject::createFromInstance
-     *
-     * @throws \LogicException
-     */
-    public static function createFromNode(
```

Comment:
Note: dropped because it is not an instance method, so an override does not really serve any purpose. The user would get a `ReflectionClass` instance anyway.

Reply:
Also: not really a BC break since this method was only throwing